README.md

```diff
@@ -75,10 +75,9 @@ After you're finished please remove all the comments and instructions!
 - **[Styling css with colors, borders, paddings, margins & sizes, etc]**
 _ **[React, Redux & javascript functions, modules, ES6 syntax, variables, React testing, etc]**
 
-## 🚀 Live Demo <a href="https://react-redux-capstone-q0vaxnxec-atril33.vercel.app/">Live Link</a>
+## 🚀 Live Demo <a href="https://maurice.onrender.com/">Live Link</a>
```

Comment:
- [ ] Good job with the project presentation, however, your missed in your presentation to highlight a piece of the code (perhaps from your code editor) that you find interesting or that you were proud of making. Let's include this missing piece in the video presentation.